### PR TITLE
Adds a disconnect wallet button

### PR DESCRIPTION
**Pull request by keithbro-imx**
Just adds a disconnect button when connected

Screen Shot 2023-10-11 at 7 25 58 am

### CODE REVIEW

File: src/components/CollectionButton/CollectionButton.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ interface CollectionButtonProps extends UnstyledButtonProps {
   name: string;
   contractAddress: string;
   updatedAt: string;
-  icon?: React.ReactNode;
+  icon?: JSX.Element;
```
Comment (keithbro-imx) [inline src/components/CollectionButton/CollectionButton.tsx:33]: Fixes the type error

File: src/components/NFTCard/NFTCard.tsx [hunk 2]
```diff
@@ -167,6 +167,7 @@ export function NFTCard({
         orderComponents: orderComponents,
         orderHash: orderHash,
         orderSignature: signature,
+        makerFees: [],
```
Comment (keithbro-imx) [inline src/components/NFTCard/NFTCard.tsx:170]: Was missing

File: src/components/PriceInfo/PriceInfo.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ export function PriceInfo({ fees, listing }: PriceInfoProps) {
   const royalty = fees.find((fee) => fee.type === orderbook.FeeType.ROYALTY);
   const protocol = fees.find((fee) => fee.type === orderbook.FeeType.PROTOCOL);
   const taker: orderbook.Fee = {
-    type: orderbook.FeeType.MAKER_MARKETPLACE,
+    type: orderbook.FeeType.MAKER_ECOSYSTEM,
```
Comment (keithbro-imx) [inline src/components/PriceInfo/PriceInfo.tsx:15]: Fix type error

File: src/contexts/Web3ProviderContext.tsx [hunk 4]
```diff
@@ -1,10 +1,14 @@
 import React, { createContext, useEffect, useState } from "react";
-import { Web3Provider } from "@ethersproject/providers";
+import { Web3Provider, ExternalProvider, JsonRpcFetchFunc } from "@ethersproject/providers";
 import { providers } from "ethers";
 
+interface Ethereum extends ExternalProvider, JsonRpcFetchFunc {
```
Comment (keithbro-imx) [inline src/contexts/Web3ProviderContext.tsx:5]: This way fixes the type error

**Review by lfportal — APPROVED** [review]

**Pull request merged**